defender/forta_large_delegations/serverless.yml

```diff
@@ -32,43 +32,56 @@ defender:
 
 functions:
   autotask-1:
-    name: ${self:custom.name} Serverless Autotask
-    path: autotask-1
+    name: Forta Large Delegations Filter
     trigger:
-      type: schedule
-      frequency: 525600000
+      type: webhook
     paused: false
+    path: 'autotask-1'
 
 resources:
   Resources:
     policies:
       policy-1:
         gas-price-cap: 1000
         whitelist-receivers:
-          - "0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2"
+          - '0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2'
         eip1559-pricing: true
-
+    notifications:
+     security-alerts:
+        type: discord
+        name: SecurityAlerts
+        config:
+          url: >-
+            https://discord.com/api/webhooks/974392977583669348/vD_9OBLHB0eYf0loRNXe5Eu9uR4Y5DVngxgtQ-Dde6ac36Zob7jqb56hlamf1dBWDqdy
```

Comment:
Please burn and recreate these discord channels if used by other components.  Please refactor to use secrets so not to leak in github.